doc/manifest/schema/1.0.0/defaultLocale.md

```diff
@@ -0,0 +1,204 @@
+# Windows Package Manager
+## Manifest Schema v1.0.0 Default Locale File
+
+All Windows Package Manager manifests in the Microsoft community repository are submitted using YAML syntax. A JSON schema is provided to aid authoring these files in editors, and in the other tooling related to the Windows Package Manager. This document provides detailed information regarding the usage of the YAML keys in the [default locale](https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.defaultLocale.1.0.0.json) file for multi-file manifests. Please review the [Manifest Specification](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md) if you are not familiar with this file.
+
+## Fields
+### PackageIdentifier
+<details>
+ <summary>The package unique identifier</summary>
+
+ #### Required Field
+
+ This key is the unique identifier for a given package. This value is generally in the form of `Publisher.Package`. It is case sensitve, and this value must match the folder structure under the partition directory in GitHub.
+
+</details>
+
+### PackageVersion
+<details>
+ <summary>The package version</summary>
+
+ #### Required Field
+
+ This key represents the version of the package. It is related to the specific release this manifests targets. In some cases you will see a perfectly formed [semantic](https://semver.org) version number, and in other cases you might see something different. These may be date driven, or they might have other characters with some package specific meaning for example.
+
+ The Windows Package Manager client uses this version to determine whether or not an upgrade for a package is available. In some cases, packages may be released with a marketing driven version, and that causes trouble with the `winget upgrade` command. 
+
+ The current best practice is to use the value reported in Add / Remove Programs when this version of the package is installed. In some cases, packages do not report a version resulting in an upgrade loop or other unwanted behavior.
+</details>
+
+#### PackageLocale
+<details>
+  <summary>The package meta-data locale</summary>
+
+  #### Required Field
+  
+  This key represents the locale for package meta-data. The format is BCP-47. This value identifies the language for meta-data to be displayed to a user when no locale file matching their preferences is available. The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules for this file.
+ </details>
```

Comment:
Could you link to a proper page which contains the BCP-47 codes?
Currently i'm taking these as reference
https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/available-language-packs-for-windows
https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/default-input-locales-for-windows-language-packs